src/pages/AboutUs/TeamMemberCards.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { MDBCard, MDBCardBody, MDBIcon } from 'mdbreact';
+import PropTypes from 'prop-types';
+
+function TeamMemberCards({ member }) {
+  return (
+    <MDBCard
+      className=" animate transition duration-500 ease-in-out  
+                        transform hover:-translate-y-1 hover:scale-110  content-start max-w-xs m-5 bg-blue-100 "
+    >
+      <img
+        style={{ width: '9rem', marginTop: '-25%' }}
+        className=" rounded-circle img-fluid z-depth-1 mx-auto"
+        src={member.member.pic}
```

Comment:
try to find a way not to use member.member twice 
and use destructure here to make it better